var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -118,6 +118,9 @@ class Openblas(MakefilePackage):
     # See https://github.com/spack/spack/issues/19932#issuecomment-733452619
     conflicts('%gcc@7.0.0:7.3.99,8.0.0:8.2.99', when='@0.3.11:')
 
+    # See https://github.com/xianyi/OpenBLAS/issues/3074
+    conflicts('%gcc@:10.1.99', when='@0.3.13 target=ppc64le:')
```

Comment:
Just curious, what does the `:` in `target=ppc64le:` mean?
`ppc64le` and supersets of that architecture?

Reply:
Alternatively, we could apply this patch: https://github.com/xianyi/OpenBLAS/pull/3075